src/shared/api/generated/CBioPortalAPI.ts

```diff
@@ -349,6 +349,8 @@ export type Mutation = {
         'referenceAllele': string
 
         'refseqMrnaId': string
+        
+        'rsid': string
```

Comment:
This is automatically generated same as with genome nexus. So can't make changes manually in this file